integration-tests/ws-security-policy-fips/src/test/java/io/quarkiverse/cxf/it/security/policy/PolicyTestUtils.java

```diff
@@ -14,4 +21,39 @@ public static RestAssuredConfig restAssuredConfig() {
                 "client-truststore-password"));
     }
 
+    public static boolean isFipsEnabled() throws IOException {
+        return java.security.Security.getProvider("SunPKCS11-NSS-FIPS") != null;
+
+        //rhel specific
+        //        return "1".equals(Files.readString(Paths.get("/proc/sys/crypto/fips_enabled"), StandardCharsets.UTF_8));
```

Comment:
I think that solution with security provider is more general.

On the other hand, the rhel specific solution is quite faster.

I'm inclined to use a provider for detection, but I can change the solution, if anybody prefers different one,